tests/test/transport/test_point_to_point.cpp

```diff
@@ -160,6 +160,13 @@ TEST_CASE_METHOD(PointToPointClientServerFixture,
     REQUIRE(receivedDataB == sentDataB);
     REQUIRE(receivedDataC == sentDataC);
 
+    // Clear local records
+    broker.resetThreadLocalCache();
+
+    // Clear broker
+    broker.clear();
+    REQUIRE(broker.getIdxsRegisteredForGroup(groupId).empty());
+
```

Comment:
I would expect to see a stand-alone test for this change that confirms that the ptp broker is cleared when we perform a scheduler reset.